performance_regression_gradual_grow_throughput.py

```diff
@@ -0,0 +1,236 @@
+from enum import Enum
+from collections import defaultdict
+
+import json
+# from sdcm.send_email import Email
+from performance_regression_test import PerformanceRegressionTest
+from sdcm.results_analyze import ThroughputLatencyGradualGrowPayloadPerformanceAnalyzer
+
+
+class CSPopulateDistribution(Enum):
+    GAUSS = "gauss"
+    UNIFORM = "uniform"
+
+
+class PerformanceRegressionGradualGrowThroughutTest(PerformanceRegressionTest):  # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.NUM_THREADS = 500  # pylint: disable=invalid-name
+        self.CLUSTER_SIZE = self.params.get("n_db_nodes")  # pylint: disable=invalid-name
+        self.REPLICATION_FACTOR = 3  # pylint: disable=invalid-name
+        self.DATASET_SIZE = 3000 * 1024 * 1024 * 1024  # pylint: disable=invalid-name
```

Comment:
is it a 3TB dataset?
please add to a comment

Reply:
Added